NEWS.md

```diff
@@ -32,7 +32,7 @@ values based on user-defined pairing (#944)
 from a BDS dataset, e.g. ADLB. (#943)
 
 ## Updates of Existing Functions
-
```

Comment:
This isn't a change to function just to the template.  I think we can not include this in the news as it is not super important to the users.  Please remove.